### PR TITLE
test(findMany): add failing test for querying by null

### PR DESCRIPTION
**Pull request by kettanaito**
- Fixes #186

### CODE REVIEW

File: test/model/findMany.test.ts [hunk 1]
```diff
@@ -76,3 +76,26 @@ test('returns an empty array when not found matching entities', () => {
...
+  })
+
+  const john = db.user.create({ id: 'john' })
+  db.user.create({ id: 'katy', organizationId: 'org-1' })
+  const clark = db.user.create({ id: 'clark' })
+
+  const users = db.user.findMany({
+    where: {
+      organizationId: {
+        equals: null,
```
Comment (kettanaito) [inline test/model/findMany.test.ts:95]: Querying nullable fields by `null` is a legal operation.

**Comment by kettanaito** [comment]
## Insights

I suspect that the type produced by `Value` is not correct:

https://github.com/mswjs/data/blob/d32f15461fc0ef915da6709de62e9e9156b8a234/src/glossary.ts#L211-L212

First of all, it misses the `| null` union type. But even when added, it still remains just a plain model value type without the `null` union. I have no idea why this happens, neither I have the tools to debug this with the current types complexity of Data. 

Eventually, a `null` check must be introduced to the `queryTypes.ts` to produce a proper query type:

https://github.com/mswjs/data/blob/d32f15461fc0ef915da6709de62e9e9156b8a234/src/query/queryTypes.ts#L88-L89

**Comment by kettanaito** [comment]
## Update

1.0 most likely fixes this. The task is now to see if this test exists in the new test suites and if it doesn't, add it.

**Comment by kettanaito** [comment]
## Released: v1.0.1 🎉

This has been released in v1.0.1.

- 📄 [**Release notes**](https://github.com/mswjs/data/releases/tag/v1.0.1)
- 📦 [View on npm](https://www.npmjs.com/package/@msw/data/v/1.0.1)

Get these changes by running the following command:

```
npm i @msw/data@latest
```

---

_Predictable release automation by [Release](https://github.com/ossjs/release)_.
